apps/charterafrica/src/components/RichText/RichText.js

```diff
@@ -100,7 +100,9 @@ const RichText = React.forwardRef(function RichText(props, ref) {
     return null;
   }
   return (
-    <React.Fragment ref={ref}>{serialize(elements, other)}</React.Fragment>
+    <LineClampedRichTypography ref={ref} {...other}>
+      {serialize(elements, other)}
+    </LineClampedRichTypography>
```

Comment:
Why this change?

Reply:
If you paste a long content with paragraphs and have lineClamp property, I'd end up with something like this.

![image](https://user-images.githubusercontent.com/39160236/221593343-f6ced7ef-533d-4180-a5eb-772ed7ad99f3.png)